docs/releases/0.0.30-beta.md

```diff
@@ -0,0 +1,21 @@
+## [0.0.30-beta](https://github.com/cloudzero/cloudzero-charts/compare/v0.0.28...v0.0.30-beta) (2024-11-12)
```

Comment:
@wreckedred and @beckilee, would you mind reviewing these Beta release notes?

Reply:
Thank you - lgtm